src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Blank.txt

```diff
@@ -104,12 +102,3 @@ false
 
 >> Blank(
 Errors: Error 6-6: Expected an operand. The formula or expression expects a valid operand. For example, you can add the operand '2' to the expression ' 1 +_' so that the result is '3'. Or, you can add the operand "there" to the expression '"Hi "& _ ' so that the result is 'Hi there'.|Error 6-6: Unexpected characters. The formula contains 'ParenClose' where 'Eof' is expected.
```

Comment:
Is there a particular reason that the shared test cases include specific JS error text?  Should we have a more generic error indication here and allow specific engines to override to include their exact error?